crates/deterministic-wasi-ctx/src/clocks.rs

```diff
@@ -1,4 +1,6 @@
-use cap_std::time::{Duration, Instant, SystemTime};
+use std::time::Duration;
```

Comment:
Why not https://docs.rs/cap-primitives/latest/cap_primitives/time/struct.Duration.html ?

Reply:
@jeffcharles, I moved to use this but I think it's just exported from the rust lib.